.travis.yml

```diff
@@ -2,9 +2,12 @@ language: python
 sudo: false
 python:
     - 2.7
+    - 3.5
+    - 3.6
+    - 3.7-dev
```

Comment:
Getting 3.7 takes this:
```yaml
matrix:
    include:
        - python: "3.7"
          dist: xenial
          sudo: true
```

Reply:
thanks!

Reply:
Just FYI, it's fine to mix `python:` and `matrix`. From zope.schema:

```yaml
language: python
sudo: false
python:
  - 2.7
  - 3.4
  - 3.5
  - 3.6
  - pypy
  - pypy3
matrix:
    include:
        - python: "3.7"
          dist: xenial
          sudo: true
```